src/components/PostPageDetail/index.js

```diff
@@ -2,7 +2,15 @@ import {useNavigation} from '@react-navigation/core';
 import {useRoute} from '@react-navigation/native';
 import moment from 'moment';
 import * as React from 'react';
-import {Dimensions, Keyboard, ScrollView, StatusBar, StyleSheet, View} from 'react-native';
+import {
+  Dimensions,
+  Keyboard,
+  ScrollView,
+  StatusBar,
+  StyleSheet,
+  View,
+  useWindowDimensions
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been modified to include `useWindowDimensions` from `react-native`. This is a good practice as it allows for responsive design based on the window's dimensions. However, please ensure that all other imported modules are being used in the code.
```diff
- import {Dimensions, Keyboard, ScrollView, StatusBar, StyleSheet, View} from 'react-native';
+ import {
+   Dimensions,
+   Keyboard,
+   ScrollView,
+   StatusBar,
+   StyleSheet,
+   View,
+   useWindowDimensions
+ } from 'react-native';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->